scripts.js

```diff
@@ -2,113 +2,113 @@
 
 // Colors
 // Selecting all the color boxes with swatch class, instead of selecting each one separetly.
-var boxes = document.querySelectorAll(".swatch");
+var boxes = document.querySelectorAll(".swatch")
 
 // Hex codes
 // Selecting all the text with p tag inside the box-details class, instead of selecting each one separetly.
-var hexes = document.querySelectorAll(".box-details > p");
+var hexes = document.querySelectorAll(".box-details > p")
 
 // Locks
 // Selecting all the icon span with material-symbols-outlined class, instead of selecting each one separetly.
-var locks = document.querySelectorAll(".material-symbols-outlined");
+var locks = document.querySelectorAll(".material-symbols-outlined")
 
-// delete query Selector
-savedPaletteSection = document.querySelector(".saved-palettes")
+// Side Display
+var asideSection = document.querySelector(".palettes")
+var savedPaletteSection = document.querySelector(".saved-palettes")
```

Comment:
line 17 was missing a `var`
line 16 was under the `global variables`  comment section. I moved it so it would be with the other HTML variables.

Reply:
Awesome. That makes sense.